src/actions/admin.test.js

```diff
@@ -1209,4 +1209,15 @@ describe('Actions.Admin', () => {
 
         const state = store.getState();
     });
+
+    it('sendWarnMetricAck', async () => {
+        const warnMetricAck = {
+            metric1: true,
+        };
+        nock(Client4.getBaseRoute()).
+            get('/warn_metrics/ack').
```

Comment:
Shouldn't it be mocking the `post` call instead of the `get` call?

Reply:
yes it should, will update